sqlmesh/core/engine_adapter/spark.py

```diff
@@ -321,12 +321,28 @@ def _create_table_properties(
                 )
             )
 
-        for key, value in (table_properties or {}).items():
-            properties.append(exp.Property(this=key, value=value))
+        properties.extend(self.__table_properties_to_expressions(table_properties))
 
         if properties:
             return exp.Properties(expressions=properties)
         return None
 
+    def _create_view_properties(
+        self,
+        table_properties: t.Optional[t.Dict[str, exp.Expression]] = None,
+    ) -> t.Optional[exp.Properties]:
+        """Creates a SQLGlot table properties expression for view"""
+        if not table_properties:
+            return None
+        return exp.Properties(expressions=self.__table_properties_to_expressions(table_properties))
+
+    @classmethod
+    def __table_properties_to_expressions(
```

Comment:
i don't understand your naming schema, with the double underscore, can we just stick with one?

Reply:
The double underscore represents a private method to an engine adapter that is not meant to be accessed by potential children and does not override anything of a parent.